test/e2e/util.go

```diff
@@ -1156,6 +1160,10 @@ func RunRC(config RCConfig) error {
 			Diff(oldPods, pods).Print(util.NewStringSet())
 			return fmt.Errorf(errorStr)
 		}
+		//TODO: Convert this warning to failure after #10030 is fixed.
+		if len(pods) > config.Replicas {
+			LogfPrefix("WARN", "POD overshoot occured %d (actual pods) > %d (replicas) !!! See https://github.com/GoogleCloudPlatform/kubernetes/issues/10030. This is not necessarily a failure, but hopefully it will be repaired soon.", len(pods), config.Replicas)
+		}
```

Comment:
True, in the long run this would be an error and the test would fail but right now we are accepting that an overshoot can occur and expect that the system will recover.  What if the system doesn't recover?  The test might (depending on what the failure case looks like) continue to run forever with nothing to time it out.  Until we make this a hard failure I think we need to ensure the test won't run forever.